ADHDataView/DESCRIPTION

```diff
@@ -0,0 +1,20 @@
+Package: ADHDataView
+Type: Package
+Title: Provide Shaped Data from ADH
+Version: 1.0.1
+Author: OSIsoft LLC
```

Comment:
are we keeping the author as OSIsoft on our samples for now?

Reply:
I made that assumption since we are keeping OSIsoft in the license files, my guess is that we would change this when we actually rename the entire Github organization